rotkehlchen/tests/api/test_pnl_csv.py

```diff
@@ -160,14 +146,13 @@ def test_history_export_download_csv(
         tempzipfile.write_bytes(response.content)
         with zipfile.ZipFile(tempzipfile, 'r') as zip_ref:
             zip_ref.extractall(extractdir)
-        assert_csv_export_response(response, extractdir, is_download=True, unicode_check=True)
+        assert_csv_export_response(response, extractdir, is_download=True)
```

Comment:
you can remove the logic for `unicode_check` also in `assert_csv_export_response`